integration_test/support/session_case.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Wallaby.Integration.SessionCase do
```

Comment:
Not sure if `Session` is still the best name here? I'm also at a loss for a better one, it's more or less a feeling...